tests/lava/proc/conv/test_utils.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+import unittest
+import numpy as np
+from lava.proc.conv import utils
+
+try:
+    import torch
+    import torch.nn.functional as F
+    compare = True
+    # in this case, the test compares against torch ground truth
+except ModuleNotFoundError:
+    compare = False
+    # in this case, the test only checks for error during
+    # utils.conv calculation
+
+
+class TestConv(unittest.TestCase):
+    def test_conv(self):
+        """Test convolution implementation"""
```

Comment:
Do we have any method of testing the behavior of Conv when Torch is not available? I feel as though there should be some basic unit tests with weight mats/connectivity structures of specialized forms to ensure that inputs are being summed properly, even in the absence of Torch.